src/ol_infrastructure/substructure/keycloak/__main__.py

```diff
@@ -455,21 +455,51 @@
         },
         opts=resource_options,
     ),
+    # Map Moira group membership to superset role
+    # ol-eng-data -> superset_admin
+    keycloak.AttributeToRoleIdentityMapper(
+        "ol-data-saml-superset-admin-ol-eng-data",
+        realm=ol_data_platform_realm.id,
+        attribute_friendly_name="mitMoiraMemberOf",
+        identity_provider_alias=ol_data_platform_touchstone_saml_identity_provider.alias,
+        attribute_value="ol-eng-data",
+        role="ol-superset-client.superset_admin",
+        extra_config={
+            "syncMode": "FORCE",
+        },
+        opts=resource_options,
+    ),
+    # Map Moira group membership to superset role
+    # ol-eng-finance -> superset_alpha
+    keycloak.AttributeToRoleIdentityMapper(
+        "ol-data-saml-superset-alpha-ol-eng-finance",
+        realm=ol_data_platform_realm.id,
+        attribute_friendly_name="mitMoiraMemberOf",
+        identity_provider_alias=ol_data_platform_touchstone_saml_identity_provider.alias,
+        attribute_value="ol-eng-finance",
+        role="ol-superset-client.superset_alpha",
```

Comment:
Is this assuming that the client role already exists? Should we add code for managing creation of that state?

Reply:
Did it cause the role to be created?

Reply:
Yeah the roles were created and the mapping seemed to have the correct client role associated with it

Reply:
Great! 👍